Week3/homework/fortuneTeller.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+{
+    function tellFortune(){
+        
+        return "You will be a " + jobs[Math.floor(Math.random() * jobs.length)] + " in " + locations[Math.floor(Math.random() * locations.length)] + ", and married to " + partnerNames[Math.floor(Math.random() * partnerNames.length)] + " with " + numChildren[Math.floor(Math.random() * numChildren.length)] + " kids."
```

Comment:
nit: you can also split this also multiple lines to make it easier to read. Though nice work, this code works fine!